code/game/antagonist/antagonist_equip.dm

```diff
@@ -6,7 +6,7 @@
 	// This could use work.
 	if(flags & ANTAG_CLEAR_EQUIPMENT)
 		for(var/obj/item/thing in player.contents)
-			player.drop_from_inventory(thing)
+			player.drop_from_inventory(thing,enable_animations = FALSE)
```

Comment:
`enable_animations`?

Reply:
Leftover code from when this PR used to have the animation element put in.

Reply:
It'll have to be removed then as it'll cause runtimes.

Reply:
Lohikar's commentis correct. Dew it.